integration/testrunner/grpc.go

```diff
@@ -127,6 +127,7 @@ func testGRPCEndpoint(ctx context.Context, t *testing.T, g *GRPC) {
 	t.Run("hello_req_unary_success", func(t *testing.T) {
 		t.Parallel()
```

Comment:
We should probably inject our test logger into this context so that any logs created by the client are printed when tests fail:

```
ctx := logging.WithContext(context.Background(), logging.TestLogger(t))
```

Reply:
Any logs that are logged by extracting the logger from the given context. They will only print if `-v` is given or if the specific test fails.